root/server/components.js

```diff
@@ -54,6 +54,9 @@ module.exports = {
   'elections/NotFound': require('../elections/NotFound'),
   'elections/Show': require('../elections/Show'),
   'event/NotFound': require('../event/NotFound'),
+  'genre/GenreCreate': require('../genre/GenreCreate'),
+  'genre/GenreDelete': require('../genre/GenreDelete'),
+  'genre/GenreEdit': require('../genre/GenreEdit'),
```

Comment:
Verb should probably come first in component name.

Reply:
Doesn't it make sense to keep the same GenreX format as all others? I mean, I do see the point that CreateGenre etc is a bit more acceptable English, but it would feel odd to me to have all other pages as GenreX and these as XGenre.

Reply:
> Doesn't it make sense to keep the same GenreX format as all others?

No, it doesn’t. :wink:

> I mean, I do see the point that CreateGenre etc is a bit more acceptable English,

Indeed, it is a name not a path.

> but it would feel odd to me to have all other pages as GenreX and these as XGenre.

On the contrary, these are editing actions whereas the others are just displaying data. This is the naming convention already used in practice even though it has not been formalized.

Reply:
Fair enough, changed!

Reply:
Thanks.

Now it feels odd to me to have `GenreEditForm` instead of `EditGenreForm`, but most probably that is just me.